client/src/components/Home.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Button from '@material-ui/core/Button';
+import Icon from '@material-ui/core/Icon';
+import Grid from '@material-ui/core/Grid';
+import { withStyles } from '@material-ui/core/styles';
+
+const styles = theme => ({
+  button: { margin: theme.spacing.unit },
+  rightIcon: { marginLeft: theme.spacing.unit },
+});
+
+const Home = ({ classes }) => (
+  <Grid container direction="row" justify="center" alignItems="center">
+    <Button className={classes.button} variant="contained" color="primary" size="large">
+      Play
+      <Icon className={classes.rightIcon}>send</Icon>
+    </Button>
+    <Button className={classes.button} variant="contained" color="secondary" size="large">
+      Find a Friend
+    </Button>
+  </Grid>
+);
+
+Home.propTypes = {
+  classes: PropTypes.object.isRequired, // eslint-disable-line react/forbid-prop-types
```

Comment:
Here you can use `shape()`. Read [here](https://reactjs.org/docs/typechecking-with-proptypes.html#proptypes) for more info